packages/node-resolve/README.md

```diff
@@ -141,6 +141,17 @@ customResolveOptions: {
 }
 ```
 
+### `rootDir`
+
+Type: `String`<br>
+Default: `process.cwd()`
+
+Root directory to resolve modules from. Used when resolving entrypoint imports, and when resolving deduplicated modules. Useful when executing rollup in a package of a monorepository.
+
+```
+rootDir: path.join(process.cwd(), '..', '..')
```

Comment:
I think it would be helpful to have a comment indicating this refers to a parent directory.
```suggestion
// Set the root directory to be the parent folder
rootDir: path.join(process.cwd(), '..')
```